edit.go

```diff
@@ -108,15 +110,24 @@ func edit(name string, v *vaulted.Vault) {
 		printVariables(v)
 		printAWS(v, false)
 		printSSHKeys(v)
+		printDuration(v)
 
-		input := readMenu("\nEdit vault: [a,s,v,?,q]: ")
+		input := readMenu("\nEdit vault: [a,s,v,d,?,q]: ")
 		switch input {
 		case "a":
 			aws(v)
 		case "s":
 			sshKeysMenu(v)
 		case "v":
 			variables(v)
+		case "d":
+			dur := readValue("Duration (in minutes): ")
+			duration, err := strconv.Atoi(dur)
```

Comment:
What if we used `time.ParseDuration` instead? (we'd need to do a little to indicate the format in the prompt above, maybe something like "(e.g. 15m)" and then give a more descriptive message if we failed to parse the time)